src/components/Navigation.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-navigation-drawer v-model="drawer" fixed app>
-      <UserDisplay />
+      <UserDisplay :v-if='this.$store.state.user.currentUser.firstName == "admin" '    />
```

Comment:
Is this assuming all administrative users will be named 'admin'?

Reply:
Good suggestion from doomer. We might have different user roles in the future.  

But is there really any need to check on, name, role, whatever to show the users name and picture. Ill suggest to remove the check